sdk/python/kfp/_client.py

```diff
@@ -155,29 +155,28 @@ def get_experiment(self, experiment_id=None, experiment_name=None):
     raise ValueError('No experiment is found with name {}.'.format(experiment_name))
 
   def _extract_pipeline_yaml(self, package_file):
+    def _choose_pipeline_yaml_file(file_list) -> str:
+      yaml_files = [file for file in file_list if file.endswith('.yaml')]
+      if len(yaml_files) == 0:
+        raise ValueError('Invalid package. Missing pipeline yaml file in the package.')
+      
+      if 'pipeline.yaml' in yaml_files:
+        return 'pipeline.yaml'
+      else:
+        if len(yaml_files) == 1:
+          return yaml_files[0]
+        raise ValueError('Invalid package. There is no pipeline.yaml file and there are multiple yaml files.')
```

Comment:
not sure if the error message raised here is what you are trying to express

so instead of `'Invalid package. There is no pipeline.yaml file and there are multiple yaml files.'`

shouldn't it be  'Invalid package. There are multiple yaml files.'

Reply:
No. The original behavior is still supported for now: There should either be a single YAML file (old behavior) otherwise the `pipeline.yaml` file must be present.